src/Uno.UI/Uno.UI.Wasm.csproj

```diff
@@ -84,6 +84,10 @@
 		<PRIResource Include="$(PlatformItemsBasePath)**\*.resw" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="Microsoft\UI\Input\InputSystemCursorShapeExtensions.wasm.cs" />
+	</ItemGroup>
+
```

Comment:
```suggestion
```

Reply:
Good catch, I forgot to remove this one. Thanks @Youssef1313 :)